### PR TITLE
fix(experiments): LemonViz funnel table visualisation

### PR DESCRIPTION
**Pull request by neilkakkar**
## Problem

Noticed that we don't show the basic details table on funnels in experiments anymore. Fixing that.

image

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 1]
```diff
@@ -1109,8 +1109,8 @@ export function Experiment(): JSX.Element {
                             >
                                 <div className="mt">
                                     <InsightContainer
-                                        disableHeader={experimentInsightType === InsightType.TRENDS}
-                                        disableTable={experimentInsightType === InsightType.FUNNELS}
+                                        disableHeader={true}
```
Comment (liyiy) [inline frontend/src/scenes/experiments/Experiment.tsx:1112]: Is there a particular reason we don't want to give the option of choosing a funnel display view?

Screen Shot 2022-05-13 at 9 40 01 PM

Comment (neilkakkar) [reply frontend/src/scenes/experiments/Experiment.tsx:1112]: Yep: It now also shows a date range which doesn't make a lot of sense anymore, and is confusing: you shouldn't be able to change the date ranges here. And in breakdown view, top to bottom is pretty hard to parse. And unnecessary, now that funnel view fits well

**Review by liyiy — APPROVED** [review]
